src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -10,14 +10,14 @@
     </div>
     <div class="list-group-item-text margin10">
       <div class="form-group">
-        <input [(ngModel)]="workItem.fields['system.description']" class="form-control" placeholder="Description" />
+        <input [(ngModel)]="workItem.fields['system.description']" (keypress)="eventHandler($event)" class="form-control" placeholder="Description" />
```

Comment:
This chunk of code was changed in later commit. In order to get the test running this template was re written following NG2's specification.